kubernetes/deployment.tmpl

```diff
@@ -26,3 +26,16 @@ spec:
               cpu: "500m"
           ports:
             - containerPort: 80
+---
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: thumbnailer
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: thumbnailer
+  minReplicas: 1
+  maxReplicas: 3
```

Comment:
I'd let this scale up if we get busy. Cpu / ram use is low in this pod so shouldn't put much pressure on the nodes / cluster.
```suggestion
  maxReplicas: 6
```